src/blocks/woo-portal/search/assets/scripts/vue/App.vue

```diff
@@ -424,13 +424,16 @@ watch(
 	grid-template-columns: var(--owp-search-columns);
 	padding-block: 48px;
 }
-#main-content:has(.woo-portal-detail-wrapper) {
+
+#main-content:has(.woo-portal-detail-wrapper), main:has(.woo-portal-detail-wrapper) {
   .woo-portal-search__hero, .woo-portal-search-container {
     display: none;
   }
 }
 
-#main-content .woo-portal-detail-wrapper + .woo-portal-search__hero, #main-content .woo-portal-detail-wrapper + .woo-portal-search__hero + .woo-portal-search-container {
+#main-content .woo-portal-detail-wrapper + .woo-portal-search__hero, #main-content .woo-portal-detail-wrapper + .woo-portal-search__hero + .woo-portal-search-container,
+main .woo-portal-detail-wrapper + .woo-portal-search__hero, main .woo-portal-detail-wrapper + .woo-portal-search__hero + .woo-portal-search-container
```

Comment:
Uhmm, looks to me this could be reduced? Not sure why you would need to display none this to be honest. 

Why would we load it than?

Rather see a ClassName on the element that makes it hidden.